iothub_client/samples/iotedge_downstream_device_sample/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if(NOT ${use_mqtt} AND NOT ${use_amqp} AND NOT ${use_http})
+    message(FATAL_ERROR "iotedge_downstream_device_sample being generated without protocol support")
+endif()
+
+compileAsC99()
+
+set(iotedge_downstream_c_files
+    iotedge_downstream_device_sample.c
+)
+
+IF(WIN32)
+    #windows needs this define
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+ENDIF(WIN32)
+
+#Conditionally use the SDK trusted certs in the samples
+if(${use_sample_trusted_cert})
```

Comment:
>use_sample_trusted_cert [](start = 5, length = 23)

Please remove the use_sample_trusted_cert block.  It's not really used in the .c (see some of my comments there on this) and it's actually potentially misleading because these certs don't matter for Edge transparent gateway scenario.

Reply:
Good call, fixed.